subsys/net/lib/download_client/Kconfig

```diff
@@ -11,13 +11,15 @@ if NRF_DOWNLOAD_CLIENT
 
 config NRF_DOWNLOAD_MAX_FRAGMENT_SIZE
 	int "Fragment size"
-	default 2048
+	range 512 2048 if BSD_LIBRARY && NRF_DOWNLOAD_CLIENT_TLS
+	default 2048 if BSD_LIBRARY && NRF_DOWNLOAD_CLIENT_TLS
```

Comment:
Should we add a word of explanation on why do we have this limitation?

Reply:
Yes, nice. This should go in the help text, imho.

Reply:
Added an explicit sentence in `help`.